src/overlays/actors/ovl_En_Fr/z_en_fr.c

```diff
@@ -27,10 +26,27 @@ const ActorInit En_Fr_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fr/EnFr_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fr/EnFr_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fr/EnFr_Update.s")
+void EnFr_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnFr* this = THIS;
+
+    if (Flags_GetSwitch(globalCtx, ENFR_GET_SWITCHFLAG(thisx))) {
```

Comment:
Do these `ENFR_GET_SWITCHFLAG` and `ENFR_GET_TARGETMODE` macros *require* `thisx` to match? Or could you use `&this->actor` instead? We tend to prefer the latter if it still matches, but if it's required then no worries.

Reply:
Still matches!

Reply:
Great! How about the one in Update?